src/immutable-data.md

```diff
@@ -20,7 +20,6 @@ Other immutable data types include:
 
 - `Option[t]`       : A type that is either `None` or `Some(t)`.
 - `Result[e, t]`    : A type that is either `Ok(t)` or `Err(e)`.
-- `Chain[t]`        : An immutable sequence of elements of type `t` that supports fast append.
```

Comment:
Why delete this? :) Please put it back :)

Reply:
This is deleted since `Chain` is also mentioned in the list above, so it is listed twice

Reply:
Oh. Well spotted. I must be blind :)